path.cabal

```diff
@@ -21,49 +21,49 @@ library
   hs-source-dirs:    src/
   ghc-options:       -Wall -O2
   exposed-modules:   Path, Path.Internal
-  build-depends:     base >= 4 && <5
-                   , exceptions
-                   , filepath < 1.2.0.1 || >= 1.3
-                   , template-haskell
+  build-depends:     aeson
+                   , base       >= 4.7     && < 5.0
                    , deepseq
-                   , aeson
-                   , hashable >= 1.2 && < 1.3
+                   , exceptions >= 0.6     && < 0.9
+                   , filepath   < 1.2.0.1  || >= 1.3
+                   , hashable   >= 1.2     && < 1.3
```

Comment:
> In my experience bounds on test suites mostly create needless maintenance work for package maintainers and stackage maintainers. But maybe you have other experiences?

I never had problems with them. I can be responsible for updating them.

> Compatible versions are already implicitly documented in the stack resolvers

Stack and Stackage are great, but they build on top of ecosystem, which by itself should be properly documented. Packages sholud be defined by their Cabal files, not an external entity such as Stackage's resolvers. At least that's my opinion. As I said I specify all bounds for all my projects, and I maintain about 20 of them by now. It has never been a slightest issue for me to bump versions when necessary and release new revisions.